### PR TITLE
FB_IC5: updates of the IC5 module

### LINKED ISSUE #72: Updates on IC5 module
Extending the current IC5 module by including another two dissipative ice models:
1.  the viscoelastic Robinson & Palmer ice model (i.e., the RP model in Mosig et al. (2015))
2.  the Order 3 power law proposed by Meylan et al. (2018)

### PR DESCRIPTION
**Pull request by QingxiangLiu**
# Pull Request Summary
Updates of the IC5 module

## Description
* Extended the IC5 module by including two additional effective medium ice models: 1) Robinson and Palmer (RP) model and 2) the Model with Order 3 Power law of Meylan et al. (2018)
* Minor updates of the manual for ST6 and IC2 (two typos only)

### Issue(s) addressed
* Is there an issue associated with this development (bug fix, enhancement, new feature)?
Closes #72 

### Check list  
* Is your feature branch up to date with the authoritative repository (NOAA/develop)? - YES.
* Make sure you have checked the [checklist for a developer submitting to develop](https://github.com/NOAA-EMC/WW3/wiki/Code-Management#checklist-for-a-developer-submitting-to-develop), and [updating version number](https://github.com/NOAA-EMC/WW3/wiki/Code-Management#checklist-for-updating-version-number) - YES
* Reviewers: @mentions of suggested reviewers of the proposed changes. @aliabdolali @JessicaMeixner-NOAA 

### Testing
* How were these changes tested? - Tested with the regtest ww3_tic1.1 and confirmed the codes work as expected.
* Are the changes covered by regression tests? (If not, why? Do new tests need to be added?) - Not fully.
* If a new feature was added, was a new regression test added?  - yes, the ww3_tic1.1 folder was updated with a few subfolders related to IC5 (i.e., input_IC5_M1/2/3)
* Have regression tests been run? - Only ww3_tic1.1 was tested.
* Which compiler / HPC you used to run the regression tests in the PR? - Intel compiler
* Please provide the summary output of matrix.comp (_matrix.Diff.out_, _matrixCompFull.out_ and _matrixCompSummary.out_):    
Please indicate the expected changes in the outputs ([excluding the known list of non-identical tests](https://github.com/NOAA-EMC/WW3/wiki/How-to-use-matrix.comp-to-compare-regtests-with-master#4-look-at-results)). - **matrix.base** is updated as well. Only changes in IC5-related tests are expected.

### CODE REVIEW

File: manual/eqs/ICE5.tex [hunk 1]
```diff
@@ -1,26 +1,42 @@
...
+
+Through a detailed theoretical analysis, \citet{Meylan2018} suggested under the assumption that $k_r$ does not deviate from the open-water wavenumber $k_0$ significantly, and that the attenuation rate $k_i$ is weak, the two VE models described above predict
+\begin{align}
+k_i^{EFS} &\propto \eta h_i^3 \sigma^{11},\label{eq:fspw}\\ k_i^{RP} &\propto \frac{\eta}{\rho_w g^2} \sigma^3,\label{eq:rppw}
+\end{align}
+whereas previous field measurements \citep[e.g.,][]{Meylan2018, Rogers2021} support a power law $k_i \propto \sigma^n$, with $n$ between 2 and 4. Eqs.~(\ref{eq:fspw}) and (\ref{eq:rppw}) indicate at certain regimes (i.e., $k_r \approx k_0$ and low $k_i$), $k_i$ of the EFS model is too sensitive to wave frequency and $k_i$ of the RP model shows no dependence on ice thickness.
+
+The third model included in the {\code IC5} module is based on the ``Model with Order 3 Power Law'' proposed by \citet[][their section 6.2; hereafter the M2 model]{Meylan2018}, which assumes the loss of wave energy is proportional to the horizontal ice velocity squared
times the ice thickness. The attenuation rate is given by
```
Comment (JessicaMeixner-NOAA) [inline manual/eqs/ICE5.tex]: Is the ^M in-between squared and times intentional? "squared^Mtimes the ice thickness"

**Review by JessicaMeixner-NOAA — COMMENTED** [review]
I'm not an expert on the sea-ice parts so I can't say much with respect to that.  I did notice one potential issue in the documentation that it looks like a special character was unintentionally added.   Assuming regression tests go as expected, I think this is fine, but waiting for the regtests before officially approving.

**Comment by aliabdolali** [comment]
The regtests ran successfully on Hera with intel and impi
The expected non-b4b results are
```
mww3_test_03
mww3_test_07
ww3_tp2.10
ww3_tp2.15
ww3_tp2.18
```
and new tests for this development:
```
ww3_tic1.1/./work_IC5_M1                    
ww3_tic1.1/./work_IC5_M2                     
ww3_tic1.1/./work_IC5_M3    
```              
[IC5.zip](https://github.com/NOAA-EMC/WW3/files/6599090/IC5.zip)

File: regtests/ww3_tic1.1/input_IC5_M1/ww3_grid_1000m.inp [hunk 2]
```diff
@@ -7,6 +7,11 @@ $
    F T F F F T
   60. 60. 60. 60.
 $
+$ IC5VEMOD defines which sea ice model is selected
+$ IC5VEMOD = 1 - Extended Fox and Squire model (EFS)
+$ IC5VEMOD = 2 - Robinson and Palmer model (RP)
+$ IC5VEMOD = 3 - Order 3 Power Law model of Meylan et al. (2018; M2)
+&SIC5 IC5VEMOD = 1 /
```
Comment (aliabdolali) [inline regtests/ww3_tic1.1/input_IC5_M1/ww3_grid_1000m.inp:14]: @QingxiangLiu Did you add tests for IC5VEMOD=2 and 3? or is there a reason for not adding them?

**Comment by aliabdolali** [comment]
Hi @QingxiangLiu
 could you merge the noaa/develop (there is a conflict)? 
Also, could you comment on Jessica's comment, then we are good to go.
AA

File: manual/eqs/ICE5.tex (hunk 1 above)
Comment (QingxiangLiu) [reply manual/eqs/ICE5.tex]: Thank you @JessicaMeixner-NOAA  for detecting this. This special character '^M' was unintentionally added during the copy-paste process, :)

File: regtests/ww3_tic1.1/input_IC5_M1/ww3_grid_1000m.inp (hunk 2 above)
Comment (QingxiangLiu) [reply regtests/ww3_tic1.1/input_IC5_M1/ww3_grid_1000m.inp:14]: Hi @aliabdolali,  I added tests for *IC5VEMOD = 2* and *IC5VEMOD = 3* as well in the subfolders **input_IC5_M2** and **input_IC5_M3**.

**Comment by QingxiangLiu** [comment]
> Hi @QingxiangLiu
> could you merge the noaa/develop (there is a conflict)?
> Also, could you comment on Jessica's comment, then we are good to go.
> AA

Hi @aliabdolali, thanks for your and @JessicaMeixner-NOAA's help. I just merged this branch with noaa/develop and have deleted the special character that Jessica detected.

Cheers,
Qingxiang

**Review by JessicaMeixner-NOAA — APPROVED** [review]

**Review by aliabdolali — APPROVED** [review]
@QingxiangLiu Thanks

**Pull request merged**